my-cart/src/app/shopping-cart/shopping-cart.component.scss

```diff
@@ -0,0 +1,116 @@
+@import './../sass/variables';
+
+.cart-items{
+    background-color: #e1dada;
```

Comment:
can't we use the color values from the SASS colour variables file?

Reply:
Added color variable in variable scss file